config/initializers/kaminari_config.rb

```diff
@@ -0,0 +1,10 @@
+Kaminari.configure do |config|
```

Comment:
Unused block argument - `config`. You can omit the argument if you don't care about it.